src/Controller/ROIOptionsController.py

```diff
@@ -91,3 +90,46 @@ def roi_name_selected(self, roi_name):
 
         self.draw_window.set_selected_roi_name(roi_name)
         self.draw_window.show()
+
+
+class RoiManipulateOptions(QtWidgets.QMainWindow, UIManipulateROIWindow):
+    """
+    Create the ROI Manipulate Options class based on the UI from the file in
+    View/ROI Delete Option
+    """
+    signal_roi_manipulated = QtCore.Signal(tuple)
+
+    def __init__(self, rois, dataset_rtss, roi_color):
```

Comment:
Do you mean docstrings for __init__ function ? I don't think that is necessary. Other classes in ROIOptionsController.py have docstrings for the class but not for individual function within them.

Reply:
Yes. Every function should have docstrings (including init functions) so anybody using this code in the future knows at a glance what the function does, what the parameters are and mean, and what the function returns (if anything).

For example in this init function, while the parameters `rois`, `dataset_rtss`, and `roi_color` are named well, their meanings are left ambiguous. Any future programmers have no choice but to guess exactly what should be passed into this function, which could lead to a lot of headaches if they guess wrong! Such as, what does `rois` mean? Is it a list of ROI names? A dictionary of ROIs? It could mean any number of things!

Adding docstrings also helps future programmers to understand what a function does without having to step through it line-by-line to try and figure it out. While it may not be necessary in this particular function (as it is straightforward), it is best practice to include docstrings in every function. If I recall correctly it's also part of PEP8 that every public function should have a docstring.